apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/server/RdaService.java

```diff
@@ -22,7 +22,7 @@
  */
 public class RdaService extends RDAServiceGrpc.RDAServiceImplBase {
   private static final Logger LOGGER = LoggerFactory.getLogger(RdaService.class);
-  public static final String RDA_PROTO_VERSION = "0.4";
+  public static final String RDA_PROTO_VERSION = "0.10";
```

Comment:
Is it possible to pull this value in from `proto/rda/VERSION.txt` or vice versa?